src/plugins/spark/features/ProcessView/settings/Kettle.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentSettings, PersistentPart } from '../state';
+import { defaultSettings } from '../components/getters';
+import { Coordinates } from '@/helpers/coordinates';
+import { slotLocations } from '../helpers/functional';
+
+const SLOTS: [number, number][] = [
```

Comment:
SLOTS suggests places where you CAN place other things. It is used in the opposite meaning here.